build_sdk_container_image

```diff
@@ -178,19 +178,25 @@ else
         rm "${tarball_copied}"
     fi
 
-    docker container rm "${toolchains_container}"
+    $docker container rm "${toolchains_container}"
 
-    host_ip="$(ip addr show docker0 | grep -Po 'inet \K[\d.]+')"
+    docker_interface="docker0"
+    if docker --version | grep -q -i podman; then
```

Comment:
both scripts `run_sdk_container` and `build_sdk_container_image` source `sdk_lib/sdk_container_common.sh` so we could define a common helper in this file rather than doing `docker --version | grep ...` twice.

What do you think ? :)

Reply:
ok, done